### PR TITLE
Corrected printing of footer (last_comment) in dep.pl

### PR DESCRIPTION
**Pull request by czurnieden**
As the titel suggests, a correction in the printing of the footer.

### CODE REVIEW

File: dep.pl [hunk 1]
```diff
@@ -81,9 +81,9 @@
    if ($apply == 1) {
       print {$out} << 'EOS';
 #endif
-/* ref:         \$Format:\%D$ */
-/* git commit:  \$Format:\%H$ */
-/* commit time: \$Format:\%ai$ */
```
Comment (minad) [inline dep.pl]: Could we just get rid of these footers? I think these are obsolete in git. With SVN such comments were very popular.

Comment (czurnieden) [reply dep.pl]: > With SVN such comments were very popular

No, they weren't just popular, they were absolutely needed! ;-)

But you are right, they are not really needed anymore with git. Hands up who is still using them?
\*puts hands in pockets\*

Comment (minad) [reply dep.pl]: I find them useless and distracting. They are also never expanded in my git checkouts. Probably git must be configured somehow, but I never bothered.

Comment (minad) [reply dep.pl]: I don't even know when these are meant to be expanded. For releases? If they would be changed every time that would be some unnecessary disk IO.

Comment (minad) [reply dep.pl]: But then - I wouldn't want release files to have different checksums than the files in git. I have no idea what the tags are for. Maybe I should think more before I hit enter.

**Comment by minad** [comment]
Ok to close?

**Pull request closed without merging**